### DISCUSSION TITLE
unable to get user from session after login

### DISCUSSION THREAD

**Discussion by ArkDouglas** [AxeWP/wp-graphql-headless-login#24] [Q&A]
I'm running into an issue where the session is not being returned from the user.js, when I attempt to use the Profile component no user is returned.    When querying /api/user directly there is no data user on the session data.

**Comment by justlevine** [comment]
Hey @ArkDouglas If you're able to share a codesandbox of your project, I'd be happy to take a look!

**Reply by ArkDouglas** [reply]
@justlevine 
Here is the codespace, I've never used on before so I don't know if it will work:
https://arkdouglas-glorious-adventure-jvx6pq4wqq5hppjq.github.dev/

Here's the repo:
https://github.com/ArkDouglas/wplogin

**Reply by justlevine** [reply]
Never used gh codespaces. Personally im a fan of https://codesandbox.io/

But no worries, pulling the repo now and will let you know how it goes

**Reply by ArkDouglas** [reply]
I've made a sandbox for it, but it's not posting graphql because of CORS issues.  


here's the URL:
https://codesandbox.io/p/github/ArkDouglas/wplogin/draft/eloquent-brown?file=%2FREADME.md&workspace=%257B%2522activeFileId%2522%253A%2522clbe7pbgi00067qdh1w70dxcw%2522%252C%2522openFiles%2522%253A%255B%255D%252C%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522gitSidebarPanel%2522%253A%2522COMMIT%2522%252C%2522spaces%2522%253A%257B%2522clbe7pd3b019s2e6cf64camkc%2522%253A%257B%2522key%2522%253A%2522clbe7pd3b019s2e6cf64camkc%2522%252C%2522name%2522%253A%2522Default%2522%252C%2522devtools%2522%253A%255B%257B%2522key%2522%253A%2522clbe7pd3c019t2e6cvryku9mi%2522%252C%2522type%2522%253A%2522PROJECT_SETUP%2522%252C%2522isMinimized%2522%253Afalse%257D%252C%257B%2522type%2522%253A%2522PREVIEW%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A3000%252C%2522key%2522%253A%2522clbe7pjj401jd2e6c7uvurzdq%2522%252C%2522isMinimized%2522%253Afalse%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522key%2522%253A%2522clbe7pits01gg2e6ckx1xf771%2522%252C%2522isMinimized%2522%253Afalse%257D%255D%257D%257D%252C%2522currentSpace%2522%253A%2522clbe7pd3b019s2e6cf64camkc%2522%252C%2522spacesOrder%2522%253A%255B%2522clbe7pd3b019s2e6cf64camkc%2522%255D%257D

**Reply by ArkDouglas** [reply]
it may be worth mentioning I am just trying to get a working sample.  This particular code base is not essential, so far as I tried to follow the steps outlined in the example to get it functioning.   I have done this with 4 different code bases, this one is just the default next starter template, so being as minimal as possible I figured it might be easier.

All I am looking for is a working implementation of this login, so I can learn how it is used and implemented and then implement it into my actual project by studying it.
